react-client/src/Components/App.js

```diff
@@ -135,22 +137,25 @@ function App() {
               id={domInfo.id}
               clsname={domInfo.clsname}
               parentId={domInfo.parentID}
-              parentClass={domInfo.parentClass}                
-              child={domInfo.child}                
+              parentClass={domInfo.parentClass}
               children={domInfo.children}
               top={domInfo.y}
               left={domInfo.x}
               onClose={handleRemoveDialogBox}
-            />     
+              seeMore={handleSeemore}
+              displayArr= {displayArray}
+            />
           ))}
+        
 
-          <DomMinimalDetailsWidget
+          {/* <DomMinimalDetailsWidget
```

Comment:
Why is this feature removed? Please restore

Reply:
sorry nkalimot lng  already restored